app/authentication_views.py

```diff
@@ -5,63 +5,95 @@
 from app import register_validation as validation
 from app.data.repositories import UsersRepository
 
+from app.logs.logger import logger
+
 
 # TODO add  generic error messages
 @app.route("/register", methods=["POST", "GET"])
 def register():
 
+    logger.info("Registering, route /register is called")
```

Comment:
the context for the method is 'Handling /register route'
registering context is when request method is POST
for those inside POST that could be
'Handling /register route, registering new user {username}, ...(rest of log)

Reply:
done